Looms.PoS/Looms.PoS.Application/Models/Requests/Product/CreateProductRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Looms.PoS.Application.Features.Business.Queries.GetBusinesses;
+
+namespace Looms.PoS.Application.Models.Requests.Product;
+
+public record CreateProductRequest
+{
+    public String Name { get; init; } = string.Empty;
+    public String Category { get; init; } = string.Empty;
+    public decimal Price { get; init; }
+    public decimal QuantityInStock { get; init; }
+    public String Description { get; init; } = string.Empty;
+    public Variation Variation { get; init; } = new Variation(string.Empty, null);
```

Comment:
``` suggestion
    public IEnumerable<Variation> Variation { get; init; } = []
```

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"